src/Components/NavCategoryBar/NavCategoryBar.component.js

```diff
@@ -32,19 +40,22 @@ class NavCategoryBar extends Component {
               <input type="text" placeholder="내 맘대로 끓여먹는 라면" />
               <i className="fa fa-search"></i>
             </div>
-            <Link to="/cartItems">
-              <img src={CART} alt="cart-icon" />
-              {cartItems.length && <span>{cartItems.length}</span>}
-            </Link>
+            <img
+              src={CART}
+              alt="cart-icon"
+              onClick={this.checkCartItemPageAccess}
+            />
+            {cartItems.length ? <span>{cartItems.length}</span> : ""}
```

Comment:
```suggestion
            {!!cartItems.length && <span>{cartItems.length}</span>}
```
true 조건일 때 렌더링하는 법 알고 계시죠!

Reply:
앞으로 계속 사용하겠습니다 ㅠㅠ